apps/broker-service/src/controllers/ETransfer.controller.ts

```diff
@@ -36,18 +37,8 @@ export class ETransferController extends Controller {
     @Response('200', 'The requesters unique eTransfer code')
     @Response('405', 'Invalid input')
     async eTransferInCode(@Body() request: SignedMessage) : Promise<ETransferCodeResponse> {
-      try {
         return {
           code: await GenerateCode(request)
         }
-      }
-      catch (err) {
-        console.error(err.message);
-        SendMail("TransferCode Error", JSON.stringify(err) + "\n---\n" + JSON.stringify(request));
-        return {
-          code: "TheCoin",
-          error: "server Error",
-        }
-      }
```

Comment:
although this isn't the point of this PR, our broker already has generalized error handling that does a better job than our custom setups here